Cargo.toml

```diff
@@ -7,9 +7,11 @@ authors = ["The Rust-gnome Project Developers"]
 libc = "0.1"
 glib = "^0"
 gdk = "^0"
-gtk = "^0"
 cairo-rs = "^0"
 
+[dependencies.gtk]
+features = ["gtk_3_10", "cairo_1_12"]
```

Comment:
We can't accept changes that prevent us from building the examples crate with older versions of GTK.
As this whole example seems to depend on 3.10 I suggest not touching the features in `Cargo.toml` and putting the whole logic into a submodule gated on `gtk_3_10` like this:
```Rust
extern crate glib;
extern crate gtk;

#[cfg(feature = "gtk_3_10")]
mod a {
    use gtk;
    // ...
    pub fn main() {
        // ...
    }
}

#[cfg(feature = "gtk_3_10")]
fn main() { a::main() }

#[cfg(not(feature = "gtk_3_10"))]
fn main() { }
```